File: doc/codes/csharp/examples/api/compute/modulation/radiation-example.cs
using AUTD3;
using static AUTD3.Units;

// HIDE
namespace AUTD3.DocSamples.ApiComputeModulationRadiationExample;

internal static class Sample
{
    internal static void Run()
    {
        // HIDE_END
var src = Modulation.ModulationBuffer();
Modulation.Sine(150 * Hz, new SineOption(), src);

var dst = Modulation.ModulationBuffer();

Modulation.RadiationPressure(src, dst);
        // HIDE
    }
}
// HIDE_END

File: doc/codes/rust/examples/api/compute/modulation/radiation-example.rs
use autd3_rs::units::Hz;
use autd3_rs_modulation::{SineOption, radiation_pressure, sine};

// HIDE
fn main() -> anyhow::Result<()> {
    // HIDE_END
    let mut src = Vec::new();
    sine(150 * Hz, &SineOption::default(), &mut src)?;

    let mut dst = Vec::new();

    radiation_pressure(&src, &mut dst);

    // HIDE
    Ok(())
}
// HIDE_END
